modules/core/src/main/java/org/apache/ignite/spi/deployment/DeploymentSpi.java

```diff
@@ -63,9 +63,10 @@
      * Finds class loader for the given class.
      *
      * @param rsrcName Class name or class alias to find class loader for.
+     * @param clsLdr desired class loader.
      * @return Deployed class loader, or {@code null} if not deployed.
      */
-    public DeploymentResource findResource(String rsrcName);
```

Comment:
You cannot remove a method from the public API.
Just mark it as deprecated.

Reply:
got it !